doc/2/api/controllers/user/update-mappings/index.md

```diff
@@ -0,0 +1,71 @@
+---
+code: true
+type: page
+title: updateMappings
+---
+
+# updateMappings
+
+<SinceBadge version="auto-version"/>
+
+Updates the internal user storage mapping.
+
+---
+
+## Query Syntax
+
+### HTTP
+
+```http
+URL: http://kuzzle:7512/user/_mappings
+Method: PUT
+Body:
+```
+
+```js
+{
+  "properties": {
```

Comment:
The example should illustrate the fact that developers can also change the `metadata` and `dynamic` properties like the `collection:updateMappings` API action